web/styles/modal.css

```diff
@@ -157,6 +157,22 @@
     }
 }
 
+.save-success {
+    display: inline-block;
+    vertical-align: top;
+    height: auto !important;
+    width: auto !important;
```

Comment:
We usually prefer to avoid "important". Also, isn't the default values of both width and height is "auto". so can we just remove them?

Reply:
Fixed.